### ISSUE TITLE
Configurable input

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#910]
Title says it all :) Now that we have a great settings manager, it's time to let the user redefine the input keys (and I will finally be able to play with ZQSD instead of WASD ;)).

Nota: a nice-to-have feature when implementing this would be a function that returns the current key for a given action, so that we don't have e.g. to hardcode key names in the help window or a future tutorial.

**Comment by Bertram25** [comment]
> Nota: a nice-to-have feature when implementing this would be a function that returns the current key for a given action, so that we don't have e.g. to hardcode key names in the help window or a future tutorial.

Good idea. :) And since we're using CEGUI, we can even return image of special keys and use them inline.
